client/components/Editors/Table/columns.tsx

```diff
@@ -34,6 +35,7 @@ export function createColumns(
     },
   }
 
+  const theme = useTheme()
```

Comment:
Hi @guergana, I removed it in #549, because it's not allowed in React:

```
chunk-ZFKBY7OL.js?v=f0261a53:521 Warning: Do not call Hooks inside useEffect(...), useMemo(...), or other built-in Hooks. You can only call Hooks at the top level of your React function. For more information, see https://reactjs.org/link/rules-of-hooks
    at TableEditor (http://localhost:8080/components/Editors/Table/index.tsx:30:11)
```

We need to find another way to share theme information here I guess

Reply:
@roll oh, sorry, I thought you removed it because you didn't need it anymore.

Reply:
@roll Please check again, now I am passing it as a prop.

Reply:
Thanks!